### PR TITLE
Create ServiceAccount for CNV-QE to pull from registry.ci.openshift.org

### PR DESCRIPTION
**Pull request by dollierp**
We already have one defined under services/cnv-qe/ folder for legacy
registry registry.svc.ci.openshift.org.

Signed-off-by: Denis Ollier <dollierp@redhat.com>

### CODE REVIEW

**Comment by dollierp** [comment]
Hi @alvaroaleman,

Is there anything missing from my side to get this PR reviewed?

Regards,

File: clusters/app.ci/registry-access/cnv-qe/admin_rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cnv-qe-admins-binding
+  namespace: cnv-qe
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: admin
```
Comment (alvaroaleman) [inline clusters/app.ci/registry-access/cnv-qe/admin_rbac.yaml:45]: Please use `view` and `pull-secret-namespace-manager` as documented https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/

Comment (dollierp) [reply clusters/app.ci/registry-access/cnv-qe/admin_rbac.yaml]: Updated according to the instructions of the document.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**